config/crd/bases/k8s.nginx.org_virtualservers.yaml

```diff
@@ -105,8 +105,8 @@ spec:
                   routing.
                 type: boolean
               listener:
-                description: Listener references a custom http and/or https listener
-                  defined in GlobalConfiguration.
+                description: VirtualServerListener references a custom http and/or
```

Comment:
did we change a property name for VS?

Reply:
This description shouldn't have been changed. In `v1.types.go` we changed the struct `Listener` to `VirtualServerListener` since `GlobalConfiguration` also had a `Listener` struct that it used, so they were conflicting with each other.

I'll change this description back though. It should be referring to the name used in the YAML definition.

Reply:
Actually nevermind I was wrong about that. The description that is put here is taken from the comment that is put above the struct in `v1/types.go` so they have to stay in line

```
// VirtualServerListener references a custom http and/or https listener defined in GlobalConfiguration.
type VirtualServerListener struct {
	HTTP  string `json:"http"`
	HTTPS string `json:"https"`
}
```

Reply:
To answer the original question, no we didn't change the property. You will still use the keyword `listener` in the Yaml definition when using custom listeners.